client/components/select-dropdown/label.jsx

```diff
@@ -3,12 +3,17 @@
 /**
  * External Dependencies
  */
-import React from 'react'
+import React, { Component } from 'react';
 
 /**
  * Module variables
  */
-const { Component } = React;
+
+/**
+ * Prevents the event from bubbling up the DOM tree
+ * @param {SyntheticEvent} event - Browser's native event wrapper
+ * @return {void}
+ */
 const stopPropagation = event => event.stopPropagation();
 
 export default class SelectDropdownLabel extends Component {
```

Comment:
would you mind use a stateless implementation here as well? I think is a good moment to refactoring :-D